apps/code-examples/src/app/code-examples/forms/radio/icon/radio-demo.component.ts

```diff
@@ -1,11 +1,11 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 import { FormBuilder, FormGroup } from '@angular/forms';
 
 @Component({
   selector: 'app-radio-demo',
   templateUrl: './radio-demo.component.html',
 })
-export class RadioDemoComponent implements OnInit {
+export class RadioDemoComponent {
   public myForm: FormGroup;
 
   public views: any[] = [
```

Comment:
Possible to get rid of the `any`? We could create a separate interface (exported from a separate file) like we did for Lookup's code examples.

Reply:
Done